docs/extend/index.md

```diff
@@ -15,7 +15,7 @@ weight = 6
 
 Currently, you can extend Docker by adding a plugin. This section contains the following topics:
 
-* [Understand Docker plugins](/extend/plugins.md)
-* [Write a volume plugin](/extend/plugins_volume.md)
-* [Write a network plugin](/extend/plugins_network.md)
-* [Docker plugin API](/extend/plugin_api.md)
+* [Understand Docker plugins](/docs/extend/plugins.md)
+* [Write a volume plugin](/docs/extend/plugins_volume.md)
+* [Write a network plugin](/docs/extend/plugins_network.md)
```

Comment:
`/extend/plugins` without the `.md` works in our current static generator.  Unfortunately, the links won't work in GitHub